gapic/src/main/com/google/gapid/perfetto/views/StyleConstants.java

```diff
@@ -274,52 +174,142 @@ public static Colors dark() {
             DARK_TIME_HIGHLIGHT,
             DARK_TIME_HIGHLIGHT_BORDER,
             DARK_TIME_HIGHLIGHT_COVER,
-            DARK_CPU_USAGE_FILL,
-            DARK_CPU_USAGE_STROKE,
             DARK_CPU_FREQ_IDLE,
             DARK_TIMELINE_RULER,
-            DARK_COUNTER_FILL,
-            DARK_COUNTER_STROKE,
-            DARK_COUNTER_HIGHLIHGT,
             DARK_TEXT_MAIN,
             DARK_TEXT_ALT,
             DARK_TEXT_INVERTED_MAIN,
-            DARK_TEXT_INVERTED_ALT,
-            DARK_THREAD_STATE_RUNNING,
-            DARK_THREAD_STATE_RUNNABLE,
-            DARK_THREAD_STATE_IOWAIT,
-            DARK_THREAD_STATE_UNINTERRUPTILE,
-            DARK_THREAD_STATE_SLEEP,
-            DARK_THREAD_STATE_UNKNOWN,
-            DARK_MEMORY_BUFFERED_CACHED,
-            DARK_MEMORY_USED,
-            DARK_BATTERY_CURRENT_OUT,
-            DARK_BATTERY_CURRENT_IN);
+            DARK_TEXT_INVERTED_ALT);
     }
   }
 
-  private static final HSL[] MD_PALETTE = new HSL[] {
-      new HSL(4, 90, 58),
-      new HSL(340, 82, 52),
-      new HSL(291, 64, 42),
-      new HSL( 262, 52, 47),
-      new HSL(231, 48, 48),
-      new HSL(207, 90, 54),
-      new HSL(199, 98, 48),
-      new HSL(187, 100, 42),
-      new HSL(174, 100, 29),
-      new HSL(122, 39, 49),
-      new HSL(88, 50, 53),
-      new HSL(66, 70, 54),
-      new HSL(45, 100, 51),
-      new HSL(36, 100, 50),
-      new HSL(14, 100, 57),
-      new HSL(16, 25, 38),
-      new HSL(200, 18, 46),
-      new HSL(54, 100, 62),
-  };
   private static final HSL GRAY_COLOR = new HSL(0, 0, 62);
 
+  /*
+   * A color scheme based on "ADUX Dataviz Palette":
+   * https://docs.google.com/spreadsheets/d/12kGr47Y5kavpBatlu4Co61UgFh7OMY9_DMg9Sd0viHI.
+   * */
+  public static class Palette {
+    private static final HSL[] BASE_COLORS = new HSL[] {
+        new HSL(202, 25, 67),
+        new HSL(200, 98, 82),
+        new HSL(20, 90, 70),
+        new HSL(131, 65, 67),
+        new HSL(262, 51, 70),
+        new HSL(338, 70, 67),
+        new HSL(171, 74, 61),
+        new HSL(42, 50, 75),
+        new HSL(198, 100, 42),
+        new HSL(172, 74, 29),
+        new HSL(262, 70, 58),
+        new HSL(22, 76, 43),
+        new HSL(217, 50, 40),
+        new HSL(59, 51, 45),
+        new HSL(320, 53, 47),
+        new HSL(215, 100, 55),
+    };
+
+    // Note: The color names correspond to HSL values in <BASE_COLORS>. Update simultaneously.
+    public static enum Name {
+      GREY,
+      LIGHT_BLUE,
+      ORANGE,
+      GREEN,
+      LIGHT_PURPLE,
+      PINK,
+      TURQUOISE,
+      TAN,
+      PACIFIC_BLUE,
+      TEAL,
+      PURPLE,
+      DARK_ORANGE,
+      INDIGO,
+      LIME,
+      MAGENTA,
+      VIVID_BLUE,
+    }
+
+    private static final int COLOR_COUNT = BASE_COLORS.length;
+    private static final int[] LIGHTULA_OFFSETS = new int[] {
+        5, 2, 4, 5, 4, 5, 6, 3, 10, 12, 6, 9, 10, 9, 9, 7,
+    };
+    private static final int[] DARCULA_OFFSETS = new int[] {
```

Comment:
Don't use the terms "lightula" and "darcula" they are IJ/AS terms. Simply "light" and "dark" for us.